ephios/plugins/qualification_management/templates/core/qualification_form.html

```diff
@@ -0,0 +1,21 @@
+{% extends "core/settings/settings_base.html" %}
+{% load crispy_forms_filters %}
+{% load i18n %}
+
```

Comment:
Add title block. (Many templates are missing a title. So maybe remove the title from other templates to make it consistent.)

Reply:
So after clicking through the most important views, I think we are good.